### ISSUE TITLE
Python Sample Code Error

### ISSUE THREAD

**Issue by faizalrf** [scylladb/alternator-load-balancing#34]
The Sample code for the Alternator load balancing, `try1.py` and `try2.py` when executing, gives the error

```
botocore.exceptions.NoRegionError: You must specify a region.
```

The line responsible for this is 

```
    dynamodb = boto3.resource('dynamodb', endpoint_url=alternator_url, 
            aws_access_key_id='alternator', aws_secret_access_key='secret_pass')
```

Updated code with the additional `region_name='us-east-1'` fixes the issue

```
    dynamodb = boto3.resource('dynamodb', endpoint_url=alternator_url, region_name='us-east-1',
            aws_access_key_id='alternator', aws_secret_access_key='secret_pass')
```

**Comment by nyh** [comment]
This is true, this is a strange AWS SDK bug, that you need a region name even if it's not actually used (because the explicit endpoint_url overrides the region anyway). The java/README.md explains (this text isn't present in python/README.md):

> The region passed to `withRegion` does not matter (and can be any string),
> because `AlternatorRequestHandler` will override the chosen endpoint anyway,
> Unfortunately we can't just drop the `withRegion()` call, because without
> it the library will expect to find a default region in the configuration file
> and complain when it is missing.

As this text explains if you have in `~/.aws/config`
```
[default]
region = us-east-1
```
The `region_name` option will no longer be required.

But I agree it's worth changing the test to include `region_name='doesnt-matter'` or something (I think using the name doesnt-matter instead of us-east-1 is better because it emphasizes that this parameter is needed but ignored).
